src/test/java/com/amazon/ion/impl/bin/IonManagedWriter_1_1_Test.kt

```diff
@@ -43,4 +48,55 @@ internal class IonManagedWriter_1_1_Test {
         }
         assertEquals("\$ion_1_1 [\$ion_1_1]", appendable.toString().trim())
     }
+
+    private fun `transform symbol IDS`(writeValuesFn: BiConsumer<IonReader, _Private_IonWriter>) {
```

Comment:
You don't need to change anything here, but FYI, you can define an equivalent functional type like this:
```kotlin
    private fun `transform symbol IDS`(writeValuesFn: (IonReader, _Private_IonWriter) -> Unit) {
```
In this case, if you wanted to you can go a step further, and actually define it as an extension function for `_Private_IonWriter`.
```kotlin
    private fun `transform symbol IDS`(writeValuesFn: _Private_IonWriter.(IonReader) -> Unit) {
```
Which would allow it to be used on L68 like this:
```kotlin
                it.writeValuesFn(reader)
```

Reply:
Cool, thanks